backend/utilities/helpers/AzureBlobStorageHelper.py

```diff
@@ -1,27 +1,55 @@
 from typing import Optional
 from datetime import datetime, timedelta
-from azure.storage.blob import BlobServiceClient, generate_blob_sas, generate_container_sas, ContentSettings
+from azure.storage.blob import BlobServiceClient, generate_blob_sas, generate_container_sas, ContentSettings, UserDelegationKey
 from .EnvHelper import EnvHelper
+from datetime import datetime, timedelta
+from azure.identity import DefaultAzureCredential
+import os
 
 class AzureBlobStorageClient:
     def __init__(self, account_name: Optional[str] = None, account_key: Optional[str] = None, container_name: Optional[str] = None):
 
         env_helper : EnvHelper = EnvHelper()
 
-        self.account_name = account_name if account_name else env_helper.AZURE_BLOB_ACCOUNT_NAME
-        self.account_key = account_key if account_key else env_helper.AZURE_BLOB_ACCOUNT_KEY
-        self.connect_str = f"DefaultEndpointsProtocol=https;AccountName={self.account_name};AccountKey={self.account_key};EndpointSuffix=core.windows.net"
-        self.container_name : str = container_name if container_name else env_helper.AZURE_BLOB_CONTAINER_NAME
-        self.blob_service_client : BlobServiceClient = BlobServiceClient.from_connection_string(self.connect_str)
-
-
+        self.use_rbac = False
```

Comment:
Should this logic live in EnvHelper?

Reply:
We introduced an `AUTH_TYPE` in `EnvHelper` to facilitate use in other files.